### DISCUSSION TITLE
TPM2-PKCS#11 for windows 10

### DISCUSSION THREAD

**Discussion by Mohanraj37** [microsoft/TSS.MSR#96] [General]
Hello Everyone,

Do we have TPM2 source in PKCS#11 standard for windows 10?

We have found TPM2-PKCS#11 for Linux and checking the similar source support for windows 10.
https://github.com/tpm2-software/tpm2-pkcs11

Thanks,
Mohanraj K.

**Comment by Mohanraj37** [comment]
Any update on this query?

**Comment by RonaldAi** [comment]
Hi,
we have not looked into a project similar to tpm2-pkcs11 for TSS.MSR.
Since the tpm2-pkcs11 project seems to be making use of the standard TPM interface, it shouldn't be too hard to create a clone that works with TSS.MSR.

Cheers, Ron.
